applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -649,6 +649,13 @@ int unicast_server_enable(le_audio_receive_cb recv_cb, enum bt_audio_location lo
 {
 	int ret;
 	static bool initialized;
+	/* clang-format off */
```

Comment:
Yeah, clang-format doesn't do well with macros like `IF_ENABLED` :(

It's up to you guys to determine if you want to add lines like `/* clang-format off */` in your code, but otherwise you probably can just save without formatting to manually format this part here (that's what I usually do in Zephyr at least)